tests/utils/unittest_ast_walker.py

```diff
@@ -42,7 +42,7 @@ def test_check_messages(self) -> None:
         linter = self.MockLinter(
             {"first-message": True, "second-message": False, "third-message": True}
         )
-        walker = ASTWalker(linter)
+        walker = ASTWalker(linter)  # type: ignore # Can't import MockLinter. This needs a refactor
```

Comment:
The issue is that `MockLinter` doesn't extend `PyLinter`

Reply:
Yes, but I also can't type `MockLinter` here. What do you want me to do?